ironic_bug_dashboard/__init__.py

```diff
@@ -14,37 +16,26 @@
 template_path = os.path.dirname(os.path.realpath(__file__))
 aiohttp_jinja2.setup(app, loader=jinja2.FileSystemLoader(template_path))
 
+parser = argparse.ArgumentParser(description='Load configuration from commandline.')
```

Comment:
Note: argparse based approach won't work for any real ASGI-based launcher, including our own https://github.com/dtantsur/ironic-bug-dashboard/blob/master/Dockerfile. Default to environment variables?